tests/timeseries/mock_microgrid.py

```diff
@@ -119,7 +119,12 @@ async def start(self, mocker: MockerFixture) -> None:
 
         # pylint: disable=protected-access
         _data_pipeline._DATA_PIPELINE = _data_pipeline._DataPipeline(
-            ResamplerConfig(resampling_period=timedelta(seconds=self._sample_rate_s))
+            ResamplerConfig(
+                resampling_period=timedelta(seconds=self._sample_rate_s),
+                # Align to the time the resampler is created to avoid flakybess
+                # in the tests
```

Comment:
Having a value in the tests that is different every time _reduces_ flakyness??

Also you wrote `flakybess`.

Reply:
Yes. I guess the test is assuming the resampling happens relative to the time the test starts. Will fix the typo though.